qutip/tests/test_metrics.py

```diff
@@ -70,14 +72,19 @@
     import cvxpy
 except:
     cvxpy = None
+try:
+    import cvxopt
+except:
+    cvxopt = None
+
+
+# We are disregarding this Issue #484 as mkl does not appear to still be incompatible with cvxpy.
```

Comment:
While I get why you did this, the next person to read this file won't know why this comment is here.  The best place for this sort of comment is in the commit message, so the developer who looks at `git log` for the code removal will understand instead.

Reply:
Okey good to know originally it was commented ther so I thought it was the right place,